packaging_automation/templates/docker/alpine/alpine.tmpl.dockerfile

```diff
@@ -24,6 +24,8 @@ RUN apk add --no-cache \
         llvm \
         lz4-dev \
         zstd-dev \
+        libxslt-dev \
+        libxml2-dev \
```

Comment:
This was not directly related to pg 15 changes but we recently added these two lines since new alpine images does not have these packages